app/services/workers/calculate-workload-points.js

```diff
@@ -16,11 +16,25 @@ module.exports.execute = function (task) {
   var id = task.additionalData.workloadBatch.startingId
   var batchSize = task.additionalData.workloadBatch.batchSize
   var reportId = task.workloadReportId
+  var maxId = id
+
+  if (batchSize < 0) {
+    logger.error('Batchsize must be greater than 0')
+    throw (new Error('Batchsize must be greater than 0'))
+  }
+
+  var message = 'Calculating Workload Points for Workloads ' + id + ' - ' + (id + batchSize)
+
+  if (batchSize > 1) {
```

Comment:
There is no need to do the check here again if the the above change <=0 is done.

Reply:
Yea, logic has been reordered